15주차/(P)17679/(P)17649_Python_노유빈.py

```diff
@@ -0,0 +1,70 @@
+# 나머지 테스트 코드는 통과했지만 테스트 코드 10번 정확성 실패, 테스트 코드 2번 효율성 실패이다.
+
+
+
+
+# 제거되는 블럭들을 '0' 으로 치환 후 제거한 후 값이 '0'인 블럭 위의 블럭이 대치 후 list 리턴 해주기. 제거되는 블럭이 없다면 false 리턴.
+
+def remove(a):
+    list = []
+    
+    # 제거될 블럭 중 가장 왼쪽위 의 블럭의 좌표를 list 에 담기.
+    for i in range(0,len(a)-1):
+        for j in range(0,len(a[0])-1):
+            if a[i][j]==a[i+1][j] and a[i][j]==a[i][j+1] and a[i][j]==a[i+1][j+1]:
+                list.append([i,j])
+    
+    
+    # list 가 비었으면 false 리턴
+    if len(list)==0:
+        return False    
+    
+    else:
+        # 비었지 않다면 그 블럭 포함 4개의 블럭 값을 '0' 으로 치환.
+        for i in range(0,len(list)):
+            a[list[i][0]][list[i][1]]='0'
+            a[list[i][0]+1][list[i][1]]='0'
+            a[list[i][0]][list[i][1]+1]='0'
+            a[list[i][0]+1][list[i][1]+1]='0'
```

Comment:
`pythonic`하게 코드를 작성할 수 있습니다!
나아가서, `list`는 리스트를 만드는 예약어이기 때문에 변수명으로 부적절해요.
```suggestion
        for i, j in list:
            a[i][j] = '0'
```